packages/client/src/execution/vmexecution.ts

```diff
@@ -39,6 +43,8 @@ export class VMExecution extends Execution {
   private _lock = new Lock()
 
   public vm: VM
+  public merkleVm: VM | null = null
```

Comment:
Small nit and also only if you clearly agree: for me this `Vm` reads strange and I would relatively clearly prefer `VM`, so e.g. `merkleVM` (I know there is `statsVm` below e.g. as well, maybe you can update along if you agree).

So if you feel the same maybe update. 🙂 No biggie though.